### PR TITLE
Added missing sigma property on ExtendedStatsAggregation

### PR DESCRIPTION
**Pull request by jonyadamit**

### CODE REVIEW

File: src/Nest/Aggregations/Metric/ExtendedStats/ExtendedStatsAggregation.cs [hunk 1]
```diff
@@ -4,19 +4,31 @@ namespace Nest
...
 		: MetricAggregationDescriptorBase<ExtendedStatsAggregationDescriptor<T>, IExtendedStatsAggregation, T>
 			, IExtendedStatsAggregation 
-		where T : class { }
-}
\ No newline at end of file
+		where T : class
+	{
+		double? IExtendedStatsAggregation.Sigma { get; set; }
+
+		public ExtendedStatsAggregationDescriptor<T> Sigma(double? sigma) =>
```
Comment (russcam) [inline src/Nest/Aggregations/Metric/ExtendedStats/ExtendedStatsAggregation.cs]: `sigma` should be a `double` here, to adhere to conventions

**Comment by codebrain** [comment]
Thanks @jonyadamit - I have made the change that @russcam requested and merged into our master branch.

Thanks again for your contribution!

**Comment by jonyadamit** [comment]
Hi @codebrain sorry I haven't had the time to investigate why `DocGenerator` wouldn't run for me and so this got left behind. Thanks for merging this!

**Comment by Mpdreamz** [comment]
@codebrain this needs to be cherry-picked to `5.x` and `5.5` too.

**Pull request merged**